cpp/include/cudf/io/types.hpp

```diff
@@ -156,8 +157,9 @@ struct table_with_metadata {
  *
  * Used to describe buffer input in `source_info` objects.
  */
+// TODO: to be replaced by `host_span`?
```

Comment:
Probably a followup PR for this

Reply:
yup. If we do go to `host_span<byte>` (instead of `host_span<byte>`, which is equivalent to `host_buffer`) it will require some wider changes.

Reply:
This comment's position may mess up doxygen generation. Perhaps move it to some other line in the code?